datus-starrocks/tests/integration/conftest.py

```
@@ -38,4 +41,195 @@ def connector(config: StarRocksConfig) -> Generator[StarRocksConnector, None, No
+        for table in TPCH_TABLES:
+            conn.execute_ddl(f"DROP TABLE IF EXISTS `{table}`")
+
+        # Create tables
+        for ddl in TPCH_DDL:
+            conn.execute_ddl(ddl)
+
+        # Insert data
+        for data in TPCH_DATA:
+            conn.execute_insert(data)
+
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)